server/notification-providers/ntfy.js

```diff
@@ -37,12 +37,12 @@ class Ntfy extends NotificationProvider {
             let tags = [];
             let status = "unknown";
             let priority = notification.ntfyPriority || 4;
+            let downPriority = notification.ntfyPriorityDown || 5;
```

Comment:
Not shure if this would be considered breaking.
This is how the default would not change:

```suggestion
            // defaults to max(priority + 1, 5)
            let downPriority = notification.ntfyPriorityDown || (priority === 5 ? priority : priority + 1);
```